tests/cupyx_tests/scipy_tests/signal_tests/test_bsplines.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from cupy import testing
+
+import cupyx.scipy.signal  # NOQA
+
+try:
+    import scipy.signal  # NOQA
+except ImportError:
+    pass
```

Comment:
Removing these will cause *all* tests to fail because they won't be able to use `cupyx.scipy.signal` or `scipy.signal` (used as `scp.signal` later with `scp` either `cupyx.scipy` or `scipy`).